client/src/playground/console.scss

```
@@ -0,0 +1,33 @@
+:host {
+  display: flex;
+  flex-direction: column;
+  font-size: smaller;
```

```suggestion
  font-size: 0.8125rem;
```
= 13px

Smaller stacks see: https://developer.allizom.org/en-US/play?id=k7viqOqUFz%2FnVauPIJH3UvQl%2BDS213YNiVL3zNnYIbxpkWor9T7cug6Pr1mHRNsqKBRxQGM9Sk8nSzA7